doc/Command-Reference.md

```diff
@@ -1512,11 +1532,11 @@ This command displays the summary of all IPv6 bgp neighbors that are configured
   Peers 4, using 83 KiB of memory
   Peer groups 2, using 128 bytes of memory
 
-  Neighbor        V         AS MsgRcvd MsgSent   TblVer  InQ OutQ  Up/Down State/PfxRcd
-  fc00::72        4      64600    3995    5208        0    0    0 00:39:30         6400
-  fc00::76        4      64600    3994    5208        0    0    0 00:39:30         6400
-  fc00::7a        4      64600    3993    5208        0    0    0 00:39:30         6400
-  fc00::7e        4      64600    3993    5208        0    0    0 00:39:30         6400
+  Neighbor        V         AS MsgRcvd MsgSent   TblVer  InQ OutQ  Up/Down State/PfxRcd NeighborName
+  fc00::72        4      64600    3995    5208        0    0    0 00:39:30         6400 Lab-T1-01
```

Comment:
Test for below case when you have neighbor data in 2 lines: 

Neighbor        V         AS MsgRcvd MsgSent   TblVer  InQ OutQ Up/Down  State/PfxRcd   NeighborName
dfc0::20:152:1:10d
                4 64802     604     602        0    0    0 09:55:25

Reply:
Yes, the device gives output in 2 lines